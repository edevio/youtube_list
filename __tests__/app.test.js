import React from 'react';
import App from '../src/components/app';
import renderer from 'react-test-renderer';

it('Display the Correct H1', () => {
  const tree = renderer.create(
    <App></App>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

