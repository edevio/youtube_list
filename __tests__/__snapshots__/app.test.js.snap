// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display the Correct H1 1`] = `
<h1
  className="main-title"
>
  Youtube Video List
</h1>
`;
